zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1018,6 +1018,50 @@ paths:
             The decision requirements search query failed.
             More details are provided in the response body.
 
+  /decision-instances/search:
+    post:
+      tags:
+        - Decision instance
+      summary: Query decision instances (alpha)
+      description: |
+        Search for decision instances based on given criteria.
+
+        :::note
+        This endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
+        See the [Camunda 8 REST API overview](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md#query-api)
+        for further details.
+        :::
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/DecisionInstanceSearchQueryRequest"
+      responses:
+        "200":
+          description: >
+            The Decision Instance Search successful response.
```

```suggestion
            The decision instance search successful response.
```

:x: remove capitalization